lessons/basics/documentation.md

```diff
@@ -0,0 +1,331 @@
+---
+layout: page
+title: Documentation
+category: basics
+order: 10
+lang: en
+---
+
+Documenting Elixir code.
+
+## Table of Contents.
+
+- [Annotation](#annotation)
+  - [Inline Documentation](#inline-documentation)
+  - [Documenting Modules](#documenting-modules)
+  - [Documenting Functions](#documenting-functions)
+- [ExDoc](#exdoc)
+  - [Installing](#installing)
+  - [Generating Documentation](#generating-documentation)
+- [Best Practice](#best-practice)
+
+
+## Annotation
+
+How much we comment and what makes quality documentation remains a contentious issue within the programming world. However, we can all agree that documentation is important for ourselves and those working with our codebase. 
+
+Elixir treats documentation as a *first-class citizen*, offering various functions to access and generate documentation for your projects. The Elixir core provides us with many different attributes to annotate a codebase. Let's look at 3 ways:
+
+  - `#`~ For inline documentation.
```

Comment:
In place of `~` could you use `-`?  That'll be a bit more consistent with other lessons.